alert-issuetracker-jira/src/main/java/com/synopsys/integration/issuetracker/jira/cloud/JiraCloudContext.java

```diff
@@ -20,7 +20,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package com.synopsys.integration.alert.issuetracker.jira.cloud;
```

Comment:
If these are still in alert, should we keep that in the package name?

Reply:
I'm looking to move the sub projects into their own repositories this week.  That's the reason for the package change there.